package.json

```
@@ -1,45 +1,76 @@
 {
-  "name": "phantombuster-sdk",
-  "version": "0.4.0",
-  "description": "Phantombuster's SDK",
-  "main": "lib/sdk.js",
-  "scripts": {
-    "test": "echo \"Error: no tests exist\" && exit 1",
-    "prepublish": "coffee --bare -o lib/ -c src/*.coffee && sed -i '1s/^/\\#\\!\\/usr\\/bin\\/env node\\n/' lib/sdk.js",
-    "watch": "coffee --bare -w -o lib/ -c src/*.coffee"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/phantombuster/sdk.git"
-  },
-  "keywords": [
-    "phantombuster",
-    "casperjs",
-    "phantomjs",
-    "sdk",
-    "headless",
-    "chrome",
-    "scraping"
-  ],
-  "author": "Martin Tapia <martin@phantombuster.com> (https://phantombuster.com)",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/phantombuster/sdk/issues",
-    "email": "martin@phantombuster.com"
-  },
-  "homepage": "https://github.com/phantombuster/sdk",
-  "preferGlobal": true,
-  "bin": {
-    "phantombuster": "lib/sdk.js"
-  },
-  "devDependencies": {
-    "coffee-script": "1.12.x"
-  },
-  "dependencies": {
-    "cson": "3.0.x",
-    "is-my-json-valid": "2.12.x",
-    "needle": "0.10.x",
-    "node-watch": "^0.6.2",
-    "yargs": "3.27.x"
-  }
+	"name": "phantombuster-sdk",
+	"version": "0.4.0",
```

Even if there is no behavior change, the deps are different, the built code is different and the package "scripts" are different. WDYT?
```suggestion
	"version": "0.4.1",
```

---

I think it should be the v1.0, but I wait for @Drax35c2db to tell me how I should bump the version and upgrade the package